src/TestingUtils/Microsoft.AspNetCore.Testing/src/RepeatContext.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading;
+
+namespace Microsoft.AspNetCore.Testing
+{
+    public class RepeatContext
+    {
+        private static AsyncLocal<RepeatContext> _current = new AsyncLocal<RepeatContext>();
```

Comment:
Open to suggestions about this. We have tests that test repeat, so we want this to be exposed somewhere. We could also just dumb down that test to use a dictionary or some-such and then we don't need this.

Reply:
I didn't create it so I don't want to ruin it.

Reply:
Sure, that's fine.